### PR TITLE
add no_render param

### PR DESCRIPTION
**Pull request by zstan**
hi, we need to have some way to get _not rendering_ pure string requests from airflow, can you add this PR or some modifications plz ? Thanks !

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -795,10 +795,13 @@ def log(self):
 
         title = "Log"
 
-        return self.render(
-            'airflow/ti_code.html',
-            code=log, dag=dag, title=title, task_id=task_id,
-            execution_date=execution_date, form=form)
+        if (request.args.get('no_render')):
+            return log
+        else:
```
Comment (mistercrunch) [inline airflow/www/views.py:800]: no need for an `else` and indentation

File: airflow/www/views.py [hunk 2]
```diff
@@ -1278,26 +1281,29 @@ class GraphForm(Form):
...
-            ),
-            blur=blur,
-            root=root or '',
-            task_instances=json.dumps(task_instances, indent=2),
-            tasks=json.dumps(tasks, indent=2),
-            nodes=json.dumps(nodes, indent=2),
-            edges=json.dumps(edges, indent=2),)
+        if (request.args.get('no_render')):
+	        return json.dumps(task_instances)
+        else:
```
Comment (mistercrunch) [inline airflow/www/views.py:1286]: no need for an else and indent

File: airflow/www/views.py [hunk 3]
```diff
@@ -1278,26 +1281,29 @@ class GraphForm(Form):
...
-                key=lambda x: x.__name__
-            ),
-            blur=blur,
-            root=root or '',
-            task_instances=json.dumps(task_instances, indent=2),
-            tasks=json.dumps(tasks, indent=2),
-            nodes=json.dumps(nodes, indent=2),
-            edges=json.dumps(edges, indent=2),)
+        if (request.args.get('no_render')):
+	        return json.dumps(task_instances)
```
Comment (mistercrunch) [inline airflow/www/views.py:1285]: indent is off by 4

**Comment by mistercrunch** [comment]
Seems reasonable, I'm ok merging it once my few comments get tackled. 

Not necessary but nice to have: add a "raw" button for the log rendered view

**Comment by zstan** [comment]
thank you, this is very useful (i want to write paper how we use airflow) but now in few words: we have two procedure to work with airflow, first - classic, and second - embedding into already existent work flow, thats why we need pure, not rendered logs, to have an opportunity to render them into our side. Thanks a lot ones more !

**Comment by mistercrunch** [comment]
We're cleaning up the open PRs list, please rebase within the next few days or we'll close the PR, in which case you can always reopen the PR from your rebased branch

**Pull request closed without merging**